### PR TITLE
Fixes DOCS-172 enableNullHandling query option needs to be documented

### PR DESCRIPTION
**Pull request by matthewhelmke**
WIP - waiting on final review

### CODE REVIEW

**Comment by matthewhelmke** [comment]
I did a git pull before creating the branch to work on all this, and it pulled in all but the last commit. The only changes are in the final commit here.

File: developers/advanced/null-value-support.md [hunk 1]
```diff
@@ -17,7 +17,13 @@ By default, null handling is disabled (`nullHandlingEnabled=false`) in the Table
...
 To enable basic null support (`IS NULL` and `IS NOT NULL`) and generate the null index, in the Table index configuration ([tableIndexConfig](https://docs.pinot.apache.org/configuration-reference/table#tableindexconfig-1)), set **`nullHandlingEnabled=true`**.
 
-When null support is enabled, `IS NOT NULL` and `IS NULL` evaluate to `true` or `false` according to whether a null is detected.&#x20;
+When null support is enabled, `IS NOT NULL` and `IS NULL` evaluate to `true` or `false` according to whether a null is detected.
+
+{% hint style="info" %}
+**Important**
+
+You MUST `SET enableNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableNullHandling=true` when you execute a query.
+{% endhint %}
```
Comment (kelseiv) [inline developers/advanced/null-value-support.md]: I'm curious if this applies to "basic null handling"?

@gortiz can you confirm? If lines 25-26 applies for basic null handling, what is the distinction between "basic" and "advanced" null handling?

Comment (gortiz) [reply developers/advanced/null-value-support.md]: It applies to both basic and advanced null handling.

The difference between basic and advanced is that basic null handling only supports `IS NOT NULL` and `IS NULL` predicates, while advanced null handling should be SQL compatible.

Comment (matthewhelmke) [reply developers/advanced/null-value-support.md]: Thank you, @gortiz . I'll make that clear before merging.

**Comment by matthewhelmke** [comment]
Added edits from review and fixed conflicts.

File: developers/advanced/null-value-support.md [hunk 2]
```diff
@@ -17,7 +17,13 @@ By default, null handling is disabled (`nullHandlingEnabled=false`) in the Table
 
 To enable basic null support (`IS NULL` and `IS NOT NULL`) and generate the null index, in the Table index configuration ([tableIndexConfig](https://docs.pinot.apache.org/configuration-reference/table#tableindexconfig-1)), set **`nullHandlingEnabled=true`**.
 
-When null support is enabled, `IS NOT NULL` and `IS NULL` evaluate to `true` or `false` according to whether a null is detected.&#x20;
+When null support is enabled, `IS NOT NULL` and `IS NULL` evaluate to `true` or `false` according to whether a null is detected.
+
+{% hint style="info" %}
+**Important**
+
+You MUST `SET enableNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableNullHandling=true` when you execute a query. Basic null handling only supports `IS NOT NULL` and `IS NULL` predicates. Advanced null handling adds SQL compatiblilty.
```
Comment (kelseiv) [inline developers/advanced/null-value-support.md]:
```suggestion
You MUST `SET enableNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableNullHandling=true` when you execute a query. Basic null handling supports `IS NOT NULL` and `IS NULL` predicates. Advanced null handling adds SQL compatibility.
```

File: developers/advanced/null-value-support.md [hunk 3]
```diff
@@ -65,6 +71,12 @@ To enable `NULL` handling, do the following:
 1. `To enable` null handling during ingestion, in [tableIndexConfig](https://docs.pinot.apache.org/configuration-reference/table#tableindexconfig-1), set**`nullHandlingEnabled=true`**.
 2. To enable null handling for queries, set the**`enableNullHandling`** [query option](https://docs.pinot.apache.org/users/user-guide-query/query-options).
 
+{% hint style="info" %}
+**Important**
+
+You MUST `SET enableNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableNullHandling=true` when you execute a query. Basic null handling only supports `IS NOT NULL` and `IS NULL` predicates. Advanced null handling adds SQL compatiblilty.
```
Comment (kelseiv) [inline developers/advanced/null-value-support.md]:
```suggestion
You MUST `SET enableNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableNullHandling=true` when you execute a query. Basic null handling supports `IS NOT NULL` and `IS NULL` predicates. Advanced null handling adds SQL compatibility.
```

**Pull request merged**